tests/files/fail/9f.yaml

```diff
@@ -33,4 +33,5 @@ schema:
             - type: str
               unique: True
 errors:
+  - 'value.notunique :: value: foo : /2/name : /1/name'
```

Comment:
Oh, you're right. Since the log was showing such different messages in the error log comparison I didn't notice that. I will change it and upload it again. Thanks!